engine/crates/postgres-connector-types/src/database_definition/walkers/relation.rs

```diff
@@ -41,8 +41,8 @@ impl<'a> RelationWalker<'a> {
         }
     }
 
-    /// True, if the referenced row is unique, this means there can only be at most one related row.
-    pub fn is_referenced_row_unique(self) -> bool {
+    /// True, if the referenced column(s) is (are) unique, this means there can only be at most one row on the other side of the relation.
+    pub fn is_other_side_one(self) -> bool {
```

Comment:
Renaming and rewriting the docstring because this is really not about the foreign key. Maybe `other_side_is_one()` would be a better name?